### PR TITLE
[PREVIEW] DIV-3478 Amed states for handling linking

### PR DESCRIPTION
**Pull request by alexwakeman**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: const/index.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
-  AosCompleted: 'AosCompleted',
-  AwaitingLegalAdvisorReferral: 'AwaitingLegalAdvisorReferral',
+  AosOverdue: 'AosOverdue',
   AosSubmittedAwaitingAnswer: 'AosSubmittedAwaitingAnswer',
-  DefendedDivorce: 'DefendedDivorce'
+  AmendPetition: 'AmendPetition',
+  AwaitingClarification: 'AwaitingClarification',
+  AwaitingConsideration: 'AwaitingConsideration',
+  AwaitingDecreeAbsolute: 'AwaitingDecreeAbsolute',
+  AwaitingDecreeNisi: 'AwaitingDecreeNisi',
```
Comment (qzhou-hmcts) [inline const/index.js]: this on the backend is using DNAwaiting

Comment (alexwakeman) [reply const/index.js]: Hmm in CCD config it is `AwaitingDecreeNisi` and on the Jira ticket DIV-3479, it's also `AwaitingDecreeNisi`... wouldn't it be better to make it consistent across all services?

Comment (qzhou-hmcts) [reply const/index.js]: The value is already used on DN, so I prefer keep like this

Comment (alexwakeman) [reply const/index.js]: Have updated it

**Comment by nsingamsetty** [comment]
All checks passed. Merging this PR.

**Pull request merged**